docs/specifications.md

```diff
@@ -7,7 +7,7 @@ order: 100
 
 The following versions are available, and the specific version in use will depend on your configuration.
 
-- PHP Version: 7.4, 8.0, 8.1
+- PHP Version: 7.4, 8.0, 8.1, 8.2 See [Altis compatibility chart](docs://guides/updating-php/#altis-compatibility-chart)
 - MySQL Version: 5.6, 5.7 (compatible; using AWS Aurora)
```

Comment:
Can we remove 5.6 version is no longer in use and has been fully deprecated also by AWS Aurora.

Reply:
Done. Thanks